.github/workflows/main.yml

```diff
@@ -102,7 +102,9 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
       - name: Install dependencies
-        run: pip install --upgrade -r requirements-lint.txt
+        run: |
+          pip install ansible
```

Comment:
Why is it needed to install ansible separate?

Reply:
ansible-lint doesn't depend on it (anymore), but putting it into lint.txt would pull it in other places where we want the git checkout of ansible instead.

Reply:
The correct way to deal with is to call `pip install --upgrade -r requirements-lint.txt ansible`, not 2 separate calls to pip. This allows the pip's dependency resolution to work correctly.